syntaxes/graphql.js.json

```diff
@@ -3,43 +3,40 @@
   "injectionSelector": "L:source -string -comment",
   "patterns": [
```

Comment:
This is split into four patterns now instead of one pattern that kinda tries to capture everything with a monster RegExp.